olive/common/utils.py

```diff
@@ -151,6 +151,26 @@ def flatten_dict(dictionary, stop_condition=None):  # pragma: no cover
     return result
 
 
+def get_dict_value(dictionary: dict, key: Union[str, Tuple, List[str]]):
+    """Get value from a nested dictionary."""
```

Comment:
Maybe update the function name to indicate that it is used to get value from nested dictionary.

Reply:
Sure. I can update it in the follow up. Maybe to something like `get_nested_dict_value`?